openlcs/packages/views.py

```diff
@@ -987,3 +904,96 @@ class ComponentViewSet(ModelViewSet, PackageImportTransactionMixin):
     """
     queryset = Component.objects.all()
     serializer_class = ComponentSerializer
+
+    def list(self, request, *args, **kwargs):
+        """
+        Get a list of components.
+
+        ####__Request__####
+
+            curl -X GET -H "Content-Type: application/json" \
+-H "Authorization: Token your_token" \
+%(HOST_NAME)s/%(API_PATH)s/components/
+
+        ####__Response__####
+
+            HTTP 200 OK
+            Content-Type: application/json
+            [
+                {
+                    "id": 5,
+                    "source": {
+                        "id": 5,
+                        "packages": [],
```

Comment:
Still exist packages ?

Reply:
ahh these docstring needs to be updated. will fix.